google-search-results.php

```diff
@@ -263,7 +263,11 @@ function query($path, $output, $q) {
     
     $api = new RestClient([
         'base_url' => "https://serpapi.com",
-        'user_agent' => 'google-search-results-php/1.3.0'
+        'user_agent' => 'google-search-results-php/1.3.0',
+        'curl_options' => [
+          CURLOPT_SSL_VERIFYHOST => 0,
+          CURLOPT_SSL_VERIFYPEER => 0,
+        ]
```

Comment:
Is this safe for users of SerpApi SDK?

Reply:
> Is this safe for users of SerpApi SDK?

No it's not required and it won't effect the users.